### DISCUSSION TITLE
Argo workflow with istio service mesh on AKS 1.19

### DISCUSSION THREAD

**Discussion by kalyanteja19** [argoproj/argo-workflows#7741] [Q&A]
I am running Argo workflow with istio service mesh on AKS. 

After AKS upgrade to 1.19, now all workflows pods are showing error status, through job is completing successfully.

In AKS 1.18, there is no such error.

We are using istio 1.9.8

**Comment by terrytangyuan** [comment]
Are you using the same version of Argo Workflows? I think this is a question to AKS upgrading but not Argo specific.

**Reply by kalyanteja19** [reply]
@terrytangyuan, Thank you. Argo workflow is not working with AKS 1.19, so needing some suggestion from your team. As I understand that there are some annotations/ podspecpatch which will be like workaround, but those did not work for me. Please find below example blog for reference.
https://argoproj.github.io/argo-workflows/sidecar-injection/

**Reply by terrytangyuan** [reply]
@alexec Any suggestions here?

**Reply by alexec** [reply]
Need more information.

**Reply by kalyanteja19** [reply]
@alexec , Could you please suggest me on the additional information you need?

**Reply by kalyanteja19** [reply]
@alexec , Could you please suggest me on the additional information you need?
